src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,25 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+import { Comments } from '../../types/Comments';
+
+type Props = {
+  comments: Comments[];
+  id: number;
```

Comment:
Rename the 'id' prop to 'postId' for better clarity and understanding of its purpose.